xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-picker/xwiki-platform-attachment-picker-ui/src/main/resources/Attachment/Picker/Code/CKEditorImagePlugin.xml

```diff
@@ -134,6 +134,29 @@
     picker.on('xwiki:attachmentGalleryPicker:unselected', function() {
       imageSelector.updateSelectedImageReferences([], $(this));
     });
+    // We extend the attachment picker to allow quick form submission
+    let results = picker.find('.attachmentPickerResults');
```

Comment:
General remake, I tend to prefer `const` over `let` when the value is not modified elsewhere.
```suggestion
    const results = picker.find('.attachmentPickerResults');
```